Fixed #35631 -- Extract quality value from Accept header (#18415)
django/django: docs/releases/5.2.txt

Requests and Responses
~~~~~~~~~~~~~~~~~~~~~~

> * The :func:`django.http.HttpRequest.get_preferred_type` can be used to query the
>   preferred media type the client accepts.

Reviewer: ```suggestion
* The new :func:`django.http.HttpRequest.get_preferred_type` method can be used
  to query the preferred media type the client accepts.
```

>   most preferred media type the client accepts.

Reviewer 2: ```suggestion
 preferred media type the client accepts.
```

* Adding :attr:`.EmailMultiAlternatives.alternatives` is now only supported via
  the :meth:`~.EmailMultiAlternatives.attach_alternative` method.

> * ``HttpRequest.accepted_types`` is now sorted by the client's preference.

Reviewer 2: ```suggestion
* ``HttpRequest.accepted_types`` is now sorted based on the request Accept
  header.
```

* Adding :attr:`.EmailMultiAlternatives.alternatives` is now only supported via
  the :meth:`~.EmailMultiAlternatives.attach_alternative` method.

> * ``HttpRequest.accepted_types`` is now sorted by the client's preference, based
>   on the request's``Accept`` header.

Reviewer: ```suggestion

* ``HttpRequest.accepted_types`` is now sorted by the client's preference, based
  on the request's ``Accept`` header.
```
